code/game/mecha/equipment/tools/work_tools.dm

```diff
@@ -438,7 +438,7 @@
 	NC.d2 = fdirn
```

Comment:
Could you change line 437 here to use NO_DIRECTION

Reply:
added